Tests/SwiftSyntaxBuilderTest/VariableTests.swift

```diff
@@ -30,15 +30,13 @@ final class VariableTests: XCTestCase {
 
   func testVariableDeclWithExplictiTrailingCommas() {
     let buildable = VariableDecl(letOrVarKeyword: .let, bindings: [
-      PatternBinding(pattern: "a", initializer: InitializerClause(value: ArrayExpr(
-        leftSquare: .`leftSquareBracket`.withTrailingTrivia(.newlines(1)),
-        elementsBuilder: {
-        for i in 1...3 {
-          ArrayElement(
-            expression: IntegerLiteralExpr(i),
-            trailingComma: .comma.withoutTrailingTrivia().withTrailingTrivia(.newlines(1)))
-        }
-      })))
+      PatternBinding(pattern: "a", initializer: InitializerClause(
+        value: ArrayExpr(leftSquare: .`leftSquareBracket`.withTrailingTrivia(.newline)) {
+          for i in 1...3 {
+            ArrayElement(expression: "\(i)", trailingComma: .comma.withoutTrailingTrivia().withTrailingTrivia(.newline))
```

Comment:
I don’t think this transformation is technically correct. The old example generates an `IntegerLiteralExpr` while this creates an `IdentifierExpr` (which conforms to `ExpressibleAsIdentifierExpr`). Identifiers are e.g. variable names, which shouldn’t consist purely of digits. 

While both generate the same source code, the SwiftSyntax tree with the `IdentifierExpr` shouldn’t technically be valid.